lib/requester/core.js

```diff
@@ -259,8 +269,16 @@ module.exports = {
         self.ensureHeaderExists(options.headers, 'User-Agent', 'PostmanRuntime/' + version);
         self.ensureHeaderExists(options.headers, 'Accept', '*/*');
 
+
+        (_.isUndefined(options.noCacheHeader) ? true : options.noCacheHeader) &&
```

Comment:
Just `options.noCacheHeader && ...`

Reply:
We also want to send these two headers by default, if requester options or PPB options are not provided to override this behaviour.

Reply:
How can one set `noCacheHeader`? You need to first add it to requester options.

Reply:
Requester options, PPB or custom cache-control header, but if none is provided by default we want to send these two, unless somebody overrides them with custom headers or disables them with options.

Reply:
Right, but here: https://github.com/postmanlabs/postman-runtime/pull/740/files#diff-4a97c1387dbab5c3fb9f31a111b03b56R378

I see `noCacheHeader: false` but assertion checks for its existence.

Reply:
No it's correct, we are overriding the requester option by PBB, where we set it to true.